### DISCUSSION TITLE
Welcome to tuxi Discussions!

### DISCUSSION THREAD

**Discussion by bugswriter** [Bugswriter/tuxi#105] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by colonvirus** [comment]
may linux torvalds bless us

**Comment by owl4ce** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Hello! I'm Harry**, I just contribute what I can do such as documentation if I have enough time. Does not have much knowledge of shellscripting and scraping 😃

**Reply by PureArtistry** [reply]
after seeing it was you who made the gif for the readme, I thought I would take a quick look at your profile.
saw the dotfiles repo and now I know what I'm going to be spending the next few hours doing, from the scrots it looks like I'm going to find some gems in there!

**Comment by primorock** [comment]
I want to contribute but I don't know bash scripting...

**Reply by bugswriter** [reply]
Learn.

**Comment by BeyondMagic** [comment]
I'm Magic, BeyondMagic, whatever you want to call me. I try to contribute the best that I can with shell scripting, simplyfing the current script for performance or less line writing. Sometimes I try to add features or fix bugs, but those are dangerous nowadays, so I'll avoid more it.

**Comment by Genghius** [comment]
develop branch: *breaks*
Genghius:
![image](https://user-images.githubusercontent.com/62076069/108636682-bd908b00-747e-11eb-94e1-e6e981d1c5b6.png)

**Reply by bugswriter** [reply]
![image](https://user-images.githubusercontent.com/23283318/108641102-c47ac600-749d-11eb-8b4f-a420f1372d71.png)

**Reply by bugswriter** [reply]
![image](https://user-images.githubusercontent.com/23283318/108641144-099ef800-749e-11eb-942f-859e42cf7a70.png)

**Reply by bugswriter** [reply]
![image](https://user-images.githubusercontent.com/23283318/108641391-46b7ba00-749f-11eb-9283-a31f9a30c208.png)

**Reply by bugswriter** [reply]
![image](https://user-images.githubusercontent.com/23283318/108641423-75ce2b80-749f-11eb-92f9-739257b794ae.png)

**Reply by bugswriter** [reply]
![image](https://user-images.githubusercontent.com/23283318/108641488-cc3b6a00-749f-11eb-9265-8644caf46ff0.png)

**Reply by bugswriter** [reply]
![image](https://user-images.githubusercontent.com/23283318/108641590-4b30a280-74a0-11eb-82eb-ed11c9714837.png)

**Reply by BeyondMagic** [reply]
LMAO, this is pure GOLD

@Genghius

**Reply by Genghius** [reply]
> LMAO, this is pure GOLD
> 
> @Genghius

There is NOTHING flex tape cant fix!

**Reply by zachspar** [reply]
> There is NOTHING flex tape cant fix!

Except the USA's measurement system.

**Reply by Genghius** [reply]
> > There is NOTHING flex tape cant fix!
> 
> Except the USA's measurement system.

what is a USA?

**Reply by sudocanttype** [reply]
Heh, you expect us Americans to have common sense? Nahhhhhhh, we want to be different so we use the most unconventional measurements possible.

**Comment by BeyondMagic** [comment]
Can someone confirm if not only location, but language as well affects **Google Search UI**? Clone the `main`, change the **LANG** variable's value to **ja_JP** and test this query **"72^43/12(44+1)-3124"**. It doesn't give me any result.

**Reply by Genghius** [reply]
![image](https://user-images.githubusercontent.com/62076069/108725001-d603b280-751d-11eb-99a2-18b12ef58e2f.png)
No problems by my side, but if there were any, we would just need to act like japan does not exist(just like USA).

**Comment by zachspar** [comment]
I'm Zach, I go by Ziggy. I like open source software and any terminal UI that makes my life easier. Oh and did I mention I run Arch as my daily driver 😂. Would do Gentoo, but building from source wastes my time.

**Reply by owl4ce** [reply]
> I'm Zach, I go by Ziggy. I like open source software and any terminal UI that makes my life easier. Oh and did I 
mention I run Arch as my daily driver . Would do Gentoo, but building from source wastes my time.

Don't try gentoo, if you don't have a good reason because it will only waste your time being productive. I ~~use~~ install gentoo (when unemployed) because of its interest in customizing flags for shrinking file binary sizes, performance optimization, and portage. Moreover, LFS rose to the top to become an independent distribution. That's crazy.

Now I fall in love and use it as a daily driver. UwU

**Reply by zachspar** [reply]
Did lfs, was so happy to boot and finally understand, at a high level, what makes Linux tick. Recommend it for learning purposes , but maintenance can be tough for daily driver use cases. I just love arch because it's so easy and maintainable.

**Reply by sudocanttype** [reply]
_laughs in ubuntu_

**Reply by owl4ce** [reply]
**bloated**

**Reply by sudocanttype** [reply]
shhhhh thats besides the point

**Reply by Genghius** [reply]
i use arch btw

**Comment by sudocanttype** [comment]
Oh, looks like we're introducing ourselves. I'm sudocanttype and I'm an avid programmer. That being said, I have no idea how my program works, it just kinda does. Most likely because my programming style is chugging 5 cans of red bull at 1 am and repeatedly bashing my head against my keyboard until something works. Also, I hate regex so unbelievably much,

**Reply by owl4ce** [reply]
> Oh, looks like we're introducing ourselves. I'm sudocanttype and I'm an avid programmer. That being said, I have no idea how my program works, it just kinda does. Most likely because my programming style is chugging 5 cans of red bull at 1 am and repeatedly bashing my head against my keyboard until something works. Also, I hate regex so unbelievably much,

https://en.wikipedia.org/wiki/Malbolge 😄

**Reply by BeyondMagic** [reply]
> https://en.wikipedia.org/wiki/Malbolge :smile:

Malbolge is so high level that I confused it with Python

**Comment by PureArtistry** [comment]
Hi all, I'm Dave.
I'm a hobby coder, I mostly enjoy writing shell scripts and little utilities (or hacking on other people's) to create my perfect desktop environment.

**Comment by Genghius** [comment]
didn't realize people were introducing themselves...
I'm Genghius, the wierdo that names his PR's the cure to all diseases yet there needed to be more than 3 of them.

**Comment by Genghius** [comment]
https://www.youtube.com/watch?v=ezqO5BTWsZc
Yet another tuxi video uwu.

**Reply by sudocanttype** [reply]
niceeeeeeee. woah this project is a little over a month old now. wack
